styles/colors.less

```diff
@@ -98,18 +89,18 @@
 @sand-checkbox-disabled-selected-text-color: var(--sand-checkbox-disabled-selected-text-color, @sand-disabled-selected-color);
 @sand-formcheckboxitem-focus-text-color: var(--sand-formcheckboxitem-focus-text-color, @sand-focus-text-color);
 @sand-item-disabled-focus-bg-color: var(--sand-item-disabled-focus-bg-color, #e6e6e6);
-@sand-keyguide-bg-color-rgb: var(--sand-keyguide-bg-color-rgb, 55, 58, 65); // #373a41
+@sand-keyguide-bg-color: var(--sand-keyguide-bg-color, rgb(var(--sand-keyguide-bg-color-rgb, 55, 58, 65))); // #373a41
+@sand-keyguide-bg-color-alpha: var(--sand-keyguide-bg-color-alpha, 0.9);
```

Comment:
As we discussed let's rename `@sand-keyguide-bg-color-alpha` to `@sand-keyguide-bg-color` with color css function and `@sand-keyguide-bg-color` to `@sand-keyguide-bg-base-color`.

Reply:
I updated the code and it looks good to leave @sand-keyguide-bg-color-opacity since it makes easy to change opacity in high-contrast skin variants just like other components.